README.md

```diff
@@ -30,9 +30,10 @@ Start using CVAT online for free: [cvat.ai](https://cvat.ai). Or set it up as a
 - [Installation guide](https://opencv.github.io/cvat/docs/administration/basics/installation/)
 - [Manual](https://opencv.github.io/cvat/docs/manual/)
 - [Contributing](https://opencv.github.io/cvat/docs/contributing/)
-- [Django REST API documentation](https://opencv.github.io/cvat/docs/administration/basics/rest_api_guide/)
 - [Datumaro dataset framework](https://github.com/cvat-ai/datumaro/blob/develop/README.md)
-- [Command line interface](https://opencv.github.io/cvat/docs/manual/advanced/cli/)
+- [Server API](https://opencv.github.io/cvat/docs/integration/api/)
```

Comment:
Is it still a valid link?

Reply:
I like to have these links. Maybe, we could provide a TOC section in the beginning? It will simplify navigation for users, because the current page is quite long. [Example from Datumaro](https://github.com/cvat-ai/datumaro#dataset-management-framework-datumaro):
![Screenshot from 2022-09-15 11-18-01](https://user-images.githubusercontent.com/13832349/190352774-01ba5b26-e163-4655-9227-61acdeaca948.png)
Then, the link would just navigate to the section below.